assets/js/googlesitekit/middleware/preloading.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * createPreloadingMiddleware tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import createPreloadingMiddleware from './preloading';
+
+describe( 'Preloading Middleware', () => {
```

Comment:
It would be nice if we could also add a test or two for this to show that it works as expected when used with `apiFetch` (doesn't need to do all the same things as the middleware test here but mostly to show it works in integration).